README.md

```diff
@@ -139,6 +139,11 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
 
 Note: HPE OneView' credentials are stored in clear-text. Make sure only authorized users can access this file.
 
+## SCMB notes:
+
+SCMB event listening will only be started if authentication_mode is set to `conf`. If necessary the app will request OneView to generate SCMB certs file and/or download the certs to the correct location. The certs file are: **oneview\_ca**: OneView's CA cert file located at: `certs/oneview_ca.pem`. **scmb\_cert**: OneView's SCMB Client cert file located at: `certs/oneview_scmb.pem`. **scmb\_key**: OneView's SCMB Client key file located at: `certs/oneview_scmb.key`
```

Comment:
Currently, Event Service works only when `authentication_mode` is set to `conf`. As it connects directly to HPE OneView SCMB, the toolkit will request OneView to generate...